src/main/java/com/google/starfish/services/NoteService.java

```diff
@@ -0,0 +1,42 @@
+package com.google.starfish.services;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;  
+import java.sql.ResultSet;  
+import javax.sql.DataSource;
+
+/**
+ * Service class for Notes that allows retrieval of values stored as
+ * instance variables on the Note class that aren't stored on the 
+ * notes table in sql
+ *
+ */
+public class NoteService extends GenericService {
+
+  public ResultSet getById(DataSource pool, long id) throws SQLException {
```

Comment:
Perhaps rather than overriding these methods in each child class, since you're not really adding extra functionality beyond what's in the super method, you could simply require a Table type in the GenericService constructor and save that as an instance variable referenced in the methods rather than taken in as a parameter.  And then here, you can just instantiate super with `Table.NOTES` for example.

Reply:
This makes a lot of sense! Since the table isn't changing across methods of the child service, it more clear to keep it as an instance variable in the parent class.